### PR TITLE
Felinids dislike chocolate with a passion

### PR DESCRIPTION
**Pull request by Bibby0110**
# General Documentation

### Intent of your Pull Request

Realism. Not actually harmful just makes em throw up. No chocolate for you kitty.

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Felinids don't like chocolate

# Changelog

:cl:  
tweak: Felinids/Catpeople hate chocolate.
/:cl:

### CODE REVIEW

**Comment by Xoxeyos** [comment]
Could you please add a mood debuff, if there isn't one already?

**Comment by Bibby0110** [comment]
> Could you please add a mood debuff, if there isn't one already?

They get a big one for consuming 'toxic' food already :)
